src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
The problem I can see here is the usage of `current`. If someone uses this link in a browser when using 2.3, it might get the wrong docs as current is currently 2.6. Using `2.3` is not a good idea either as it would be a nightmare to maintain over time. One thing I can think of is using a `any` (or anything else that would make more sense) version. When detecting this version, we would ask the user the version he is using and redirect to the appropriate page. What do you think?

ping @symfony/deciders